embedded/Localization/fr-FR.json

```diff
@@ -1,5 +1,5 @@
 {
-  "Options.SECTION[ExtendedRadio.ExtendedRadio.ExtendedRadioMod]": "ExtendedRadio",
+  "Options.SECTION[ExtendedRadio.ExtendedRadio.ExtendedRadioMod]": "Extended Radio",
```

Comment:
I would expect that the proper name of the mod would stay the same in all languages